examples/network/imageSelected/imageSelected.html

```diff
@@ -0,0 +1,70 @@
+<html>
+<head>
+  <title>Network | Selected/Unselected Image</title>
+
+  <script type="text/javascript" src="../../../dist/vis.js"></script>
+  <link href="../../../dist/vis-network.min.css" rel="stylesheet" type="text/css" />
+
+  <style type="text/css">
+    body {
+      font: 10pt arial;
+    }
+    #mynetwork {
+      width: 600px;
+      height: 600px;
+      border: 1px solid lightgray;
+      background-color:#333333;
```

Comment:
Can you please remove the background color here. I like it but it does not go well with the broken image.